.gitignore

```diff
@@ -265,6 +265,5 @@ pip-log.txt
 /chpl/chpl-service/logs/
 /chpl/deploy-webapp.sh
 /chpl/.recommenders/
-/chpl/chpl-api/src/main/webapp/WEB-INF/web.xml
```

Comment:
There are a bunch of /*.properties files a few lines above that are also ignored. Shouldn't they be removed from this file too?